[WARNING] No expected exception occurred in online testing (resulting in a failure) but java.lang.AssertionError occurred now.
[WARNING] Finished offline run as no further events were recorded during online test.
[WARNING] No expected exception occurred in online testing (resulting in a failure) but java.lang.AssertionError occurred now.
[WARNING] Finished offline run as no further events were recorded during online test.
